onset_maproom/maproom.py

```diff
@@ -165,7 +296,8 @@ def onset_plots(
                 color="danger",
                 dismissable=True,
             )
-            return errorFig, errorFig, alert1
+            germ_sentence = ""
+            return errorFig, errorFig, alert1, germ_sentence
```

Comment:
Didn't we decide on another PR to get rid of the alerts because the errorFigs make them redundant? I thought we had already done that; maybe that was a different situation and they're still needed here for some reason.